store/src/java/com/zimbra/cs/service/mail/SearchResponse.java

```diff
@@ -150,9 +150,10 @@ int size() {
      * @throws ServiceException error
      */
     void add(ZimbraHit zimbraHit) throws ServiceException{
-		add(zimbraHit,false);
+        add(zimbraHit,false);
+
```

Comment:
remove empty line, this change is not needed anymore

Reply:
done